cluster/mesos/docker/mesos-slave/bin/wrapdocker

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+# Copyright 2013-2014 Jérôme Petazzoni, Johan Haleby, lalyos, James Harris,
+#                     Michael Elsdörfer, Tony Hesjevik, Esben Haabendal,
+#                     Michael A. Smith
+# Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
It's fine. Let's leave it for now and I'll try to make sure this PR holds as if all these scripts were consistent, which means I'm human precommit checker.